shared/message/behaviour/state/Stability.proto

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2021 NUbots <nubots@nubots.net>
+ */
+
+syntax = "proto3";
+
+package message.behaviour.state;
+
+enum Stability {
+    // The robot has fallen on the ground
+    FALLEN = 0;
+    // The robot is not on the ground, but is not dynamically stable
+    FALLING = 1;
+    // The robot is dynamically stable, provided continued control input it should not fall
+    DYNAMIC = 2;
+    // The robot is statically stable, if control input ceased immediately it would not fall
+    STATIC = 3;
+    // The robot is standing upright on both feet
+    STANDING = 4;
+}
```

Comment:
```suggestion
    // The robot is standing upright on both feet
    STANDING = 4;
    // The robot is doing a backflip
    BACKFLIP = 5;
}
```

Reply:
Is doing a backflip considered more stable than standing? :P